app/main.py

```diff
@@ -1 +1,41 @@
-# write your code here
+class SoftwareEngineer:
+    skills = []
+
```

Comment:
`skills` variable should be an attribute of the instance not class.
read [checklist.md](https://github.com/mate-academy/py-software-engineers/blob/master/checklist.md)!